gigaword-indexer/src/main/java/edu/isi/vista/gigawordIndexer/IndexGigawordWithElasticSearch.java

```diff
@@ -127,17 +127,24 @@ private static RestHighLevelClient buildElasticSearchClient(Parameters parameter
   private static void index(RestHighLevelClient client, Path file, String indexName) throws IOException {
     log.info("Indexing {}", file.toAbsolutePath());
 
+    Iterable<ConcatenatedGigawordDocuments.Article> concatenatedGigawordDocuments;
```

Comment:
Can you add a string parameter `corpus_format` to the parameter file input to the program and have the decision about how to ingest the data come from that rather than the filename? That will be more robust, especially as we add more options. I'd suggest `gigaword` and`annotated_gigaword` as the currently-supported values.   If `annotated_gigaword` is chosen, please log a warning that offsets are not preserved and such a corpus cannot be used for the curated training process.